docstore/awsdynamodb/dynamo_test.go

```diff
@@ -86,15 +86,24 @@ func (h *harness) Close() {
 func (h *harness) MakeCollection(_ context.Context, kind drivertest.CollectionKind) (driver.Collection, error) {
 	switch kind {
 	case drivertest.SingleKey, drivertest.NoRev:
-		return newCollection(dyn.New(h.sess), collectionName1, drivertest.KeyField, "", &Options{AllowScans: true})
+		return newCollection(dyn.New(h.sess), collectionName1, drivertest.KeyField, "", &Options{
+			AllowScans:     true,
+			ConsistentRead: true,
+		})
 	case drivertest.TwoKey:
+		// For query test we don't use strong consistency mode since some tests are
+		// running on global secondary index and it doesn't support ConsistentRead.
```

Comment:
What would happen if you do set it in those tests? Ignored? Error?

Reply:
It will result in error in those cases that queries the global secondary index. I think that is the correct behavior since we don't want to make it "work" silently by ignoring the option when the user expects a `ConsistentRead`. And the error message returned is clear enough to point to this issue.

Reply:
I think this comment here is fine. I explained that in the option comment.